trustbloc/sidetree-fabric
# chore: update sidetree-core (purposes, proof files)

Included:
- handle purposes as per latest spec
- create and upload proof files
- download and verify proof files

Closes #468

Signed-off-by: Sandra Vrtikapa <sandra.vrtikapa@securekey.com>
